src/node.cc

```diff
@@ -3903,6 +3912,9 @@ static void StartNodeInstance(void* arg) {
     LoadEnvironment(env);
 
     env->set_trace_sync_io(trace_sync_io);
+    if (track_heap_objects) {
+      env->isolate()->GetHeapProfiler()->StartTrackingHeapObjects(true);
```

Comment:
true, it ideally start just after an Isolate is created, fixed to happen before env starts up